### ISSUE TITLE
Manager can only have one child with react<16 or preact

### ISSUE THREAD

**Issue by billneff79** [FezVrasta/react-popper#151]
The documentation in the README for v1 shows the `<Manager/>` component having multiple children (`<Reference/>` and `<Popper/>`).  If you are using `react` at a version less than 16, or a library like `preact` that doesn't support it, the ponyfill from `create-react-context` kicks in.  It has a [documented issue](https://github.com/jamiebuilds/create-react-context#compatibility) where it will only render properly if it has one child.

### Steps to reproduce the problem

1.  Use react<15 or preact with preact-context-provider
2.  Give Manager more than one child

### What is the expected behavior?

The children of Manager are rendered when there is more than one child

### What went wrong?

This children of Manager do not get rendered when there is more than one child

### Any other comments?

Should either fix the code so that Manager can have more than one child (preferred) or fix the documentation to point out the caveat that Manager must have a single child, like a `<div>` that wraps all of the real children

### Packages versions

- react-popper: 1.0.0-beta.5
- create-react-context: 0.2.2

**Comment by billneff79** [comment]
Yes, you can definitely wrap everything in a div.  That's certainly a valid workaround, and the one I used for my preact project.  It would just be good to note that in the main readme somewhere for folks that don't use react@16.  I burned several hours trying to figure out why my components weren't rendering at all (there is no useful error message when using preact for this issue - stuff just doesn't render at all).  Alternatively, we could use something other than `create-react-context` to provide the values in context that wouldn't have this limitation.

**Comment by FezVrasta** [comment]
There's not any way to make React 15 render multiple children from the same component, no matter what do you use to provide the context.

Good point on the docs addition, please if you have time send a PR to address it.
